ui/pages/confirmations/components/simulation-details/fiat-display.test.tsx

```diff
@@ -47,19 +47,39 @@ const mockStoreWithHidingFiatOnTestnets = configureStore()(
 
 jest.mock('../../../../hooks/useFiatFormatter');
 
+type FiatFormatterOptions = {
+  shorten?: boolean;
+  truncatedCharLimit?: number;
+  truncatedStartChars?: number;
+};
+
 describe('FiatDisplay', () => {
   const mockUseFiatFormatter = jest.mocked(useFiatFormatter);
 
   beforeEach(() => {
     jest.resetAllMocks();
-    mockUseFiatFormatter.mockReturnValue((value: number) => `$${value}`);
+    mockUseFiatFormatter.mockReturnValue(
+      (value: number, options?: FiatFormatterOptions) => {
+        const formattedValue = `${value.toLocaleString('en-US', {
+          minimumFractionDigits: 2,
+          maximumFractionDigits: 2,
+        })}`;
+        if (
+          options?.shorten &&
+          formattedValue.length > (options?.truncatedCharLimit ?? 0)
+        ) {
+          return `$${formattedValue.slice(0, options.truncatedStartChars)}...`;
+        }
+        return `$${formattedValue}`;
+      },
+    );
```

Comment:
It looks like this has been a faulty test. Mocking this method actually caused the test to not test the true output. Removing the original mock also resulted in different values and failing tests. We should remove these mocks

Reply:
Yeap, previously it was mocked, and it makes sense to remove this mock as you suggested.